browser-interface/test/unit/RestrictedActions.test.tsx

```diff
@@ -66,11 +65,16 @@ describe('RestrictedActions tests', () => {
     it('should move the player', async () => {
       setLastPlayerPosition()
       const ctx = getContextWithPermissions(PermissionItem.PI_ALLOW_TO_MOVE_PLAYER_INSIDE_SCENE)
-      const stub = sinon.stub(getUnityInstance(), 'Teleport')
+      const spy = sinon.spy(getUnityInstance(), 'Teleport')
 
       movePlayerTo({ newRelativePosition: new Vector3(8, 0, 8) }, ctx)
-
-      Sinon.assert.calledWithExactly(stub, { position: { x: 8, y: 0, z: 1624 }, cameraTarget: undefined }, false)
+      const callArguments = spy.firstCall.args
+      expect(callArguments[0].position.x).to.eq(8)
+      expect(callArguments[0].position.y).to.eq(0)
+      expect(callArguments[0].position.z).to.eq(1624)
```

Comment:
Because `position` is a Vector3, this was failing (apparently in sinon 6 the default behavior changed and the prototype of the object is checked as equality). Happy to hear different solutions to this!